src/math/Box3.js

```diff
@@ -244,29 +244,13 @@ Box3.prototype = {
 
 	containsPoint: function ( point ) {
 
-		if ( point.x < this.min.x || point.x > this.max.x ||
-				 point.y < this.min.y || point.y > this.max.y ||
-				 point.z < this.min.z || point.z > this.max.z ) {
-
-			return false;
-
-		}
-
-		return true;
+		return !(point.x < this.min.x || point.x > this.max.x || point.y < this.min.y || point.y > this.max.y || point.z < this.min.z || point.z > this.max.z);
```

Comment:
The question is still pending, but it is done with [fcf6239](https://github.com/mrdoob/three.js/pull/10064/commits/fcf6239e0bea91d0c484fe0f40a741216864bac8).